### PR TITLE
Dom mobile menu

### PR DESCRIPTION
**Pull request by Nedramrez**
- Add code for mobile menu 
- Add style to mobile menu
- Commit from partner to fix the position of nav-menu
- Add style changing features to function that opens and closes the mobile menu.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <div class="mobile-menu" id="id-for-js">
 
     </div>
```
Comment (1GORDON) [inline index.html:20]: Great job team 🤝

- [x] Kindly make sure the mobile popup options link to the intended references. When I click on about, I should be directed to the about section of the page which is not the case for your design. Kindly fix this issue and resubmit for approval.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Nedramrez & @eerapheal  ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/javascript/script.js [hunk 2]
```diff
@@ -54,7 +56,7 @@ function mobileMenu() {
   const appearMobileLink3 = document.createElement('a');
   appearMobileLink3.classList.add('mobile-link');
   appearMobileUlLi3.appendChild(appearMobileLink3);
-  appearMobileLink3.setAttribute('href', '#Contact');
```
Comment (Nedramrez) [inline assets/javascript/script.js:57]: Thank you for your friendly and professional review! 

I've fixed the issue with broken links by changing the first capital letters in the `href` attribute of an `<a>` tags.

File: assets/javascript/script.js [hunk 3]
```diff
@@ -43,7 +45,7 @@ function mobileMenu() {
   const appearMobileLink2 = document.createElement('a');
   appearMobileLink2.classList.add('mobile-link');
   appearMobileUlLi2.appendChild(appearMobileLink2);
-  appearMobileLink2.setAttribute('href', '#About');
```
Comment (Nedramrez) [inline assets/javascript/script.js:46]: Thank you for your friendly and professional review!

I've fixed the issue with broken links by changing the first capital letter in the `href` attributes of an `<a>` tags.

**Review by m05e5 — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Nedramrez  🔥 

Good job on implementing the required changes. Well Done.
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

goodjob


Now you can merge this branch 🚀 

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

_m05e5_

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nedramrez** [comment]
Thank you for your professional and friendly review!

**Pull request merged**